nvm.sh

```diff
@@ -3015,38 +3052,63 @@ nvm() {
       if [ -n "${NVM_LTS-}" ]; then
         VERSION="$(nvm_match_version "lts/${NVM_LTS:-*}")"
       elif [ -z "${PROVIDED_VERSION-}" ]; then
-        nvm_rc_version
+        # Get the .nvmrc version depending if quiet mode is on or not
+        if [ $NVM_USE_SILENT -ne 1 ]; then
+          nvm_rc_version
+        else
+          nvm_rc_version --silent
+        fi
```

Comment:
```suggestion
        NVM_USE_SILENT=${NVM_USE_SILENT:-false} nvm_rc_version
```
this way, we won't need any special handling inside `nvm_rc_version`.

Reply:
I haven't studied parameter expansions too much. So, when the ```NVM_USE_SILENT``` variable is null, ```${NVM_USE_SILENT:-false}``` would be equal to ```false```. That means that when silent mode is on then the following will execute:
```bash
1 nvm_rc_version
```
Because NVM_USE_SILENT is equal to one. And when silent mode is off the following will run:
```bash
false nvm_rc_version
```
In both scenarios, nvm_rc_version won't run. This is because false will just return an exit code of 1 and 1 is not a command in the first scenario. Could you please explain a bit more on what is happening on the following line:
```bash
NVM_USE_SILENT=${NVM_USE_SILENT:-false} nvm_rc_version
```